Broken doxygen documentation after PR #117

After merging PR #117, the generation of doxygen documentation is broken and produces a lot of warnings (see log output below).

Steps to reproduce:

1. check out commit 7be9f348debceb341aef9042abf726cd57479785
2. run `make doxy`

> /home/heinovski/src/repos/veins-heinovski/src/veins/base/phyLayer/AnalogueModel.h:40: warning: argument 'senderPos' of command @param is not found in the argument list of Veins::AnalogueModel::filterSignal(Signal *signal)=0
/home/heinovski/src/repos/veins-heinovski/src/veins/base/phyLayer/AnalogueModel.h:40: warning: argument 'receiverPos' of command @param is not found in the argument list of Veins::AnalogueModel::filterSignal(Signal *signal)=0
/home/heinovski/src/repos/veins-heinovski/src/veins/modules/analogueModel/PERModel.h:33: warning: argument 'senderPos' of command @param is not found in the argument list of Veins::PERModel::filterSignal(Signal *signal) inherited from member filterSignal at line 40 in file /home/heinovski/src/repos/veins-heinovski/src/veins/base/phyLayer/AnalogueModel.h
/home/heinovski/src/repos/veins-heinovski/src/veins/modules/analogueModel/PERModel.h:33: warning: argument 'receiverPos' of command @param is not found in the argument list of Veins::PERModel::filterSignal(Signal *signal) inherited from member filterSignal at line 40 in file /home/heinovski/src/repos/veins-heinovski/src/veins/base/phyLayer/AnalogueModel.h
/home/heinovski/src/repos/veins-heinovski/src/veins/modules/analogueModel/TwoRayInterferenceModel.h:56: warning: argument 'senderPos' of command @param is not found in the argument list of Veins::TwoRayInterferenceModel::filterSignal(Signal *signal) inherited from member filterSignal at line 40 in file /home/heinovski/src/repos/veins-heinovski/src/veins/base/phyLayer/AnalogueModel.h
/home/heinovski/src/repos/veins-heinovski/src/veins/modules/analogueModel/TwoRayInterferenceModel.h:56: warning: argument 'receiverPos' of command @param is not found in the argument list of Veins::TwoRayInterferenceModel::filterSignal(Signal *signal) inherited from member filterSignal at line 40 in file /home/heinovski/src/repos/veins-heinovski/src/veins/base/phyLayer/AnalogueModel.h

--- a/src/veins/base/phyLayer/AnalogueModel.h
+++ b/src/veins/base/phyLayer/AnalogueModel.h
@@ -42,8 +42,6 @@ public:
      * over time to the Signal.
      *
      * @param signal        The signal to filter.
-     * @param senderPos    The position of the frame sender.
-     * @param receiverPos    The position of frame receiver.
      */
     virtual void filterSignal(Signal* signal) = 0;
 

